src/contracts/libraries/GPv2Signing.sol

```diff
@@ -167,6 +169,99 @@ library GPv2Signing {
         setFreeMemoryPointer(freeMemoryPointer);
     }
 
+    /// @dev Verifies the input calldata as an EIP-1271 contract signature and
+    /// returns the address of the signer.
+    ///
+    /// The encoded signature tightly packs the following struct:
+    ///
+    /// ```
+    /// struct EncodedEip1271Signature {
+    ///     address verifier;
+    ///     uint16 signatureLength;
+    ///     bytes signature;
+    /// }
+    /// ```
+    ///
+    /// All entries are tightly packed together in this order in the encoded
+    /// calldata. `signatureLength` encodes the length of the signature bytes
+    /// in bytes. Example:
+    ///
+    /// input:    0x73c14081446bd1e4eb165250e826e80c5a523783000a00010203040506070809
+    /// decoding:   [..............verifier................][sL][.......data.......]
+    /// stride:                                          20   2         10 (0x000a)
```

Comment:
Is there any reason we show only base 10 representation of the first two but then show the hex representation in brackets for the last one?

Reply:
It should help to see that the length of data comes from the bytes above `[sL]`, that is `000a`. It's a shorter alternative to `(from the previous parameter)`.

Reply:
An important difference as well is that the size of `data` is variable and depends on the input.